### PR TITLE
Added reference for current navigation

### PR DESCRIPTION
**Pull request by joinimran**
As this article is linked with modern team site so I have updated the article accordingly. 

Problem: https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/477

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by joinimran** [comment]
Ready to go.

**Comment by mypil** [comment]
@kelleyvice-msft - PR has been copy edited and is ready for final review. This is based on issue #477.

Thank you.

File: Enterprise/navigation-options-for-sharepoint-online.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ ms.collection: Ent_O365
 ms.custom: Adm_O365
 search.appverid: SPO160
 ms.assetid: adb92b80-b342-4ecb-99a1-da2a2b4782eb
-description: "This article describes navigation options sites with SharePoint Publishing enabled in SharePoint Online. The choice and configuration of navigation significantly impacts the performance and scalability of sites in SharePoint Online."
+description: "This article describes navigation options sites with SharePoint Publishing enabled in SharePoint Online. The choice and configuration of navigation significantly impacts the performance and scalability of sites in SharePoint Online. This article is not applicable on Classic Team Sites."
```
Comment (nenonix) [inline Enterprise/navigation-options-for-sharepoint-online.md]:
```suggestion
description: "This article describes navigation options sites with SharePoint Publishing enabled in SharePoint Online. The choice and configuration of navigation significantly impacts the performance and scalability of sites in SharePoint Online. This article is not applicable to Classic Team Sites."
```

**Comment by mypil** [comment]
@joinimran - please merge the commit on the copy/edit review made by nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by joinimran** [comment]
Suggestions applied.

**Pull request merged**